content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -48,8 +53,9 @@ When the upload is completed you can [connect your board to the Science Journal
 
 You need to install the following libraries:
 
-- `Arduino_ScienceJournal`
-- `Adafruit LSM9DS1 Library` (MKR WiFi 1010 only)
+- `Arduino_ScienceKitCarrier` (If using the Nano RP2040 Connect)
```

Comment:
If using the board with the Science Carrier R3, not if using the board by itself